### PR TITLE
If the browser does not support crypto.subtle.digest, fall back to js-sha256

### PR DESCRIPTION
**Pull request by pedoc**

### CODE REVIEW

File: src/utils/auth.ts [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
 
 export const verifySignature = async (payload: AuthPayload, sign: string) => {
-  // if (Math.abs(payload.t - Date.now()) > 1000 * 60 * 5) {
-  //   return false
-  // }
-  const payloadSign = await generateSignature(payload)
-  return payloadSign === sign
-}
\ No newline at end of file
+  if (Math.abs(payload.t - Date.now()) > 1000 * 60 * 5) {
```
Comment (ddiu8081) [inline src/utils/auth.ts:28]: Before solving the issue #117, it is necessary to comment out timestamp verification.

File: src/utils/auth.ts [hunk 2]
```diff
@@ -1,26 +1,33 @@
+import { sha256 } from "js-sha256";
```
Comment (ddiu8081) [inline src/utils/auth.ts:1]: Do not change the code format, please keep **single quotes** and **no semicolons**.

**Review by ddiu8081 — CHANGES_REQUESTED** [review]

File: src/utils/auth.ts [hunk 3]
```diff
@@ -1,26 +1,33 @@
...
-  // if (Math.abs(payload.t - Date.now()) > 1000 * 60 * 5) {
-  //   return false
-  // }
-  const payloadSign = await generateSignature(payload)
-  return payloadSign === sign
-}
\ No newline at end of file
+  if (Math.abs(payload.t - Date.now()) > 1000 * 60 * 5) {
+    return false;
+  }
```
Comment (ddiu8081) [inline src/utils/auth.ts:30]: Before solving the issue #117, it is necessary to comment out timestamp verification.

Comment (pedoc) [reply src/utils/auth.ts]: fixed

File: src/utils/auth.ts (hunk 2 above)
Comment (pedoc) [reply src/utils/auth.ts:1]: fixed

**Review by ddiu8081 — APPROVED** [review]

**Pull request merged**